workitems/tokenlist/l2-token-list-v1.0-psd01.md

```diff
@@ -236,29 +238,43 @@ The tokens data element is a composite which MUST minimally contain the followin
 
 Note, that the detailed definition of the data elements in [[R2]](#r2) along with descriptions and examples are given in the schema itself below.
 
+[[R2]](#r2) testability: See suggested test fixtures for this documents' data schema below.
+
 #### **[D1]**
 All other data elements of the schema SHOULD be included in a representation of a canonical token list.
 
+[[D1]](#d1) testability: See suggested test fixtures for this documents' data schema below.
+
 #### **[CR1]>[D1]**
-if the extension data elements is used, the following data elements MUST be present in the schema representation:
+If the extension data elements is used, the following data elements MUST be present in the schema representation:
 - rootChainId
 - rootChainURI
 - rootAddress
 
 Note, that the detailed definition of the data elements in [[D1]](#d1) and [[CR1]>[D1]](#cr1d1) along with descriptions and examples are given in the schema itself below.
 
+[[CR1]>[D1]](#cr1d1) testability: See suggested test fixtures for this documents' data schema below.
+
 #### **[R3]**
 All properties in the schema identified in the description to be a Universal Resource Identifier (URI) MUST follow in their semantics [RFC 3986](#rfc3986).
 
+[[R3]](#r3) testability: As an approved standard, all requirements for [RFC 3986](#rfc3986) are testable.
+
 #### **[R4]**
 The `chainId` property MUST follow [EIP-155](#eip155) standard.
 
+[[R4]](#r4) testability: As an approved standard, all requirements for [EIP-155](#eip155) are testable.
+
 #### **[D2]**
 The `chainId` property SHOULD follow [EIP-3220](#eip3220) draft standard.
 
+[D2]](#d2) testability: The [EIP-3220](#eip3220) draft standard has test fixtures.
```

Comment:
No it doesn't, although it should. In many cases the requirements are clearly specified, so at least for those how to test is obvious.

(Note that I'll take this comment to the [EEA Crosschain Interoperability Working Group](https://entethalliance.github.io/crosschain-interoperability) developing that spec)

Reply:
addressed, see new commit @chaals